### PR TITLE
Add missing defaults for three OIDC settings

### PR DESCRIPTION
**Pull request by justincr-elastic**
For review of #86561 to fix a misspelled OIDC setting in the docs, I double checked all other settings in OpenIdConnectRealmSettings.java were present and spelled correctly in security-settings.java.

At the same time, I noticed these three OIDC settings have defaults in code which are not mentioned in the docs. This PR adds those missing defaults to the docs.

Doc PR previews are available for a short while after a build:
- https://elasticsearch_86746.docs-preview.app.elstc.co/diff

### CODE REVIEW

File: docs/reference/settings/security-settings.asciidoc [hunk 1]
```diff
@@ -1832,6 +1833,7 @@ a maximum period inactivity between two consecutive data packets). Defaults to
 Controls the behavior of the http client used for back-channel communication to
 the OpenID Connect Provider endpoints. Specifies the maximum number of
 connections allowed across all endpoints.
+Defaults to 200.
```
Comment (lockewritesdocs) [inline docs/reference/settings/security-settings.asciidoc:1836]:
```suggestion
Defaults to `200`.
```

File: docs/reference/settings/security-settings.asciidoc [hunk 2]
```diff
@@ -1840,6 +1842,7 @@ connections allowed across all endpoints.
 Controls the behavior of the http client used for back-channel communication to
 the OpenID Connect Provider endpoints. Specifies the maximum number of
 connections allowed per endpoint.
+Defaults to 200.
```
Comment (lockewritesdocs) [inline docs/reference/settings/security-settings.asciidoc:1845]:
```suggestion
Defaults to `200`.
```

**Review by lockewritesdocs — APPROVED** [review]
LGTM 🦖 Added formatting for `200`, but solid otherwise. Thanks @justincr-elastic!

File: docs/reference/settings/security-settings.asciidoc [hunk 3]
```diff
@@ -1770,6 +1770,7 @@ As per `claim_patterns.principal`, but for the _dn_ property.
 (<<static-cluster-setting,Static>>)
 The maximum allowed clock skew to be taken into consideration when validating
 id tokens with regards to their creation and expiration times.
+Defaults to `60m`.
```
Comment (ywangd) [inline docs/reference/settings/security-settings.asciidoc:1773]: I believe this is 60 seconds instead of 60 minutes:
```suggestion
Defaults to `60s`.
```

Comment (justincr-elastic) [reply docs/reference/settings/security-settings.asciidoc]: Thank you for catching my typo!

**Pull request merged**